openapi.yml

```diff
@@ -5569,6 +5596,143 @@ components:
+                    type: integer
+                visitor_id:
+                    description: Unique identifier of the `Visitor` this `Registration` belongs to
+                    type: integer
+                pre_registration_id:
+                    description: Unique identifier of the `PreRegistration` this `Registration` belongs to
+                    type: integer
+                collected_information:
+                    description: A hash with all the different types of possible pages on the `Registration` flow.
+            example:
+                id: 123123123
+                created_at: '2020-01-01'
+                visitor_id: 1
+                pre_registration_id: 1
+                collected_information:
+                    forms:
+                        -
+                            title: This is a form
+                            sequence: 1
+                            id: 1
+                            data:
+                                -
+                                    key: question_1
+                                    value: Red Wine
+                                -
+                                    key: question_2
+                                    value: White Wine
+                        -
+                            title: This is another form
+                            sequence: 3
+                            id: 10
+                            data:
+                                -
+                                    key: question_3
+                                    value: Postal Code
+                                -
+                                    key: question_4
+                                    value: V3B0K9
+                    hosts:
+                        -
+                            title: This is a host page
+                            sequence: 2
+                            id: 20
+                            data:
+                                -
+                                    fullname: Cam Wiebe
+                                    department: CTO
+                                    email: cwiebe@tg.com
+                                    profile_picture: 'https://tg.com/picture/jpg'
+                                -
+                                    fullname: Dan Souza
+                                    department: Dev
+                                    email: dsouza@tg.com
+                                    profile_picture: 'https://tg.com/picture/jpg'
+                    identities:
+                        -
+                            title: Tell us a bit about yourself
+                            id: 33
+                            sequence: 3
+                            data:
+                                email: gqueirz@tg.com
+                                company: TG
+                                fullname: Gabe Queiroz
+                                mobile: '+123123123'
```

Comment:
🌵 🌵 there's an automatic generator you can use, you don't have to type out examples for all these

Reply:
Hmm... where?

Reply:
When you click to add an example, or click to edit it, there's a `Generate` button in the top right corner of the field